instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py

```diff
@@ -350,7 +354,10 @@ def _instrument(self, **kwargs):
                 unit="seconds",
             )
 
-        if "process.runtime.gc_count" in self._config:
+        if "process.runtime.gc_count" in self._config and self._python_implementation != "pypy":
+            logger.warning(
```

Comment:
This warning should be shown if the interpreter is `PyPy` and `"process.runtime.gc_count" in self._config`.

Reply:
@ocelotl : In that case will put the logger.warn in else condition and make the code changes.

Reply:
@ocelotl : Changes done